ui/src/app/features/face-recognition/face-recognition-container.component.ts

```diff
@@ -60,8 +58,8 @@ export class FaceRecognitionContainerComponent implements OnInit, OnDestroy {
 
   recognizeFace(file: File) {
     file.size > MAX_IMAGE_SIZE ?
-      this.snackBarService.openError(
-        null, 8000, this.translate.instant('face_recognition_container.file_size_error')
+      this.snackBarService.openNotification(
```

Comment:
let's improve the `openNotification` method even further and make the `duration` variable inside of the method by default (if not specified in params) 3000 for info message and 8000 for warning and error. this will allow to call just `this.snackBarService.openNotification({messageText: 'face_recognition_container.file_size_error', type: 'error'})`